nb/documentation/interactions_with_geometry/interactions_with_geometry.nb

```diff
@@ -38,6 +59,18 @@ Arc(4)
   .view();
 
 
+
+md`
+---
+### Fuse
+
+Shapes which are grouped, assembled, or otherwise associated can be fused together to form a single solid
+
+`;
+
+Group(Arc(10), Arc(10).x(6)).fuse().view();
```

Comment:
We could simplify this as Arc(10).x(0, 6).fuse().view()

Reply:
I like that